src/quick-lint-js/parse.h

```diff
@@ -3412,8 +3449,13 @@ class parser {
 #if QLJS_HAVE_SETJMP
   bool have_unimplemented_token_jmp_buf_ = false;
   std::jmp_buf unimplemented_token_jmp_buf_;
+  bool have_parser_depth_limit_exceeded_jmp_buf = false;
+  std::jmp_buf parser_depth_limit_exceeded_jmp_buf_;
 #endif
 
+  std::uint16_t depth_ = 0;
```

Comment:
Nit: `std::uint16_t` is usually a bad idea for members of heavy-weight classes like `parser`. `std::uint_fast16_t` would be more appropriate, or just `int`.

Reply:
Done